lang/en/naas.php

```diff
@@ -17,156 +17,147 @@
 
 /**
  * Strings for component 'url'; language 'en'; branch 'MOODLE_20_STABLE'
- *
+ * English messages.
  * @package    mod_naas
- * @copyright  1999 onwards Martin Dougiamas  {@link http://moodle.com}
+ * @copyright (C) 2019  ISAE-SUPAERO (https://www.isae-supaero.fr/)
  * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
 
-$string['pluginname'] = 'Nugget';
-$string['modulename'] = 'Nugget';
-$string['modulename_help'] = 'The Moodle Nugget Plugin enables a teacher to integrate a micro-content from the NaaS Server.
-
-The teacher can allow the Nugget exercice to be attempted multiple times. A time limit may be set.
-
+// Clés générales
+$string['modulename'] = 'nugget';
+$string['modulename_help'] = '<p>The Moodle Nugget Plugin enables a teacher to integrate a micro-content from the NaaS Server. The teacher can allow the Nugget exercice to be attempted multiple times. A time limit may be set.
```

Comment:
I'm pretty sure the lang entry must be one line message.
The doc says that the lang files do not have to comply with the maximum nb of characters per line.